# This file is distributed under the same license as the Django package.
#
# Translators:
# Jiří Podhorecký <jirka.p@volny.cz>, 2024
# Tomáš Ehrlich <tomas.ehrlich@gmail.com>, 2015
# Vláďa Macek <macek@sandbox.cz>, 2015-2019
msgid ""
msgstr ""
"Project-Id-Version: django\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2024-12-03 06:32+0000\n"
"PO-Revision-Date: 2024-10-07 09:22+0000\n"
"Last-Translator: Jiří Podhorecký <jirka.p@volny.cz>, 2024\n"
"Language-Team: Czech (http://app.transifex.com/django/django/language/cs/)\n"
"Language: cs\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=4; plural=(n == 1 && n % 1 == 0) ? 0 : (n >= 2 && n "
"<= 4 && n % 1 == 0) ? 1: (n % 1 != 0 ) ? 2 : 3;\n"

#: venv/lib/python3.12/site-packages/django/contrib/postgres/apps.py:54
msgid "PostgreSQL extensions"
msgstr "Rozšíření pro PostgreSQL"

#: venv/lib/python3.12/site-packages/django/contrib/postgres/forms/array.py:17
#: venv/lib/python3.12/site-packages/django/contrib/postgres/forms/array.py:185
#, python-format
msgid "Item %(nth)s in the array did not validate:"
msgstr "Položka č. %(nth)s v seznamu je neplatná:"

#: venv/lib/python3.12/site-packages/django/contrib/postgres/forms/hstore.py:17
msgid "Could not load JSON data."
msgstr "Data typu JSON nelze načíst."

#: venv/lib/python3.12/site-packages/django/contrib/postgres/forms/hstore.py:18
msgid "Input must be a JSON dictionary."
msgstr "Vstup musí být slovník formátu JSON."

#: venv/lib/python3.12/site-packages/django/contrib/postgres/forms/ranges.py:42
msgid "Enter two valid values."
msgstr "Zadejte dvě platné hodnoty."

#: venv/lib/python3.12/site-packages/django/contrib/postgres/forms/ranges.py:44
msgid "The start of the range must not exceed the end of the range."
msgstr "Počáteční hodnota rozsahu nemůže být vyšší než koncová hodnota."

#: venv/lib/python3.12/site-packages/django/contrib/postgres/forms/ranges.py:99
msgid "Enter two whole numbers."
msgstr "Zadejte dvě celá čísla."

#: venv/lib/python3.12/site-packages/django/contrib/postgres/forms/ranges.py:105
msgid "Enter two numbers."
msgstr "Zadejte dvě čísla."

#: venv/lib/python3.12/site-packages/django/contrib/postgres/forms/ranges.py:111
msgid "Enter two valid date/times."
msgstr "Zadejte dvě platné hodnoty data nebo času."

#: venv/lib/python3.12/site-packages/django/contrib/postgres/forms/ranges.py:117
msgid "Enter two valid dates."
msgstr "Zadejte dvě platná data."

#: venv/lib/python3.12/site-packages/django/contrib/postgres/validators.py:15
#, python-format
msgid ""
"List contains %(show_value)d item, it should contain no more than "
"%(limit_value)d."
msgid_plural ""
"List contains %(show_value)d items, it should contain no more than "
"%(limit_value)d."
msgstr[0] ""
"Seznam obsahuje %(show_value)d položku, ale neměl by obsahovat více než "
"%(limit_value)d."
msgstr[1] ""
"Seznam obsahuje %(show_value)d položky, ale neměl by obsahovat více než "
"%(limit_value)d."
msgstr[2] ""
"Seznam obsahuje %(show_value)d položek, ale neměl by obsahovat více než "
"%(limit_value)d."
msgstr[3] ""
"Seznam obsahuje %(show_value)d položek, ale neměl by obsahovat více než "
"%(limit_value)d."

#: venv/lib/python3.12/site-packages/django/contrib/postgres/validators.py:25
#, python-format
msgid ""
"List contains %(show_value)d item, it should contain no fewer than "
"%(limit_value)d."
msgid_plural ""
"List contains %(show_value)d items, it should contain no fewer than "
"%(limit_value)d."
msgstr[0] ""
"Seznam obsahuje %(show_value)d položku, ale neměl by obsahovat méně než "
"%(limit_value)d."
msgstr[1] ""
"Seznam obsahuje %(show_value)d položky, ale neměl by obsahovat méně než "
"%(limit_value)d."
msgstr[2] ""
"Seznam obsahuje %(show_value)d položek, ale neměl by obsahovat méně než "
"%(limit_value)d."
msgstr[3] ""
"Seznam obsahuje %(show_value)d položek, ale neměl by obsahovat méně než "
"%(limit_value)d."

#: venv/lib/python3.12/site-packages/django/contrib/postgres/validators.py:38
#, python-format
msgid "Some keys were missing: %(keys)s"
msgstr "Některé klíče chybí: %(keys)s"

#: venv/lib/python3.12/site-packages/django/contrib/postgres/validators.py:39
#, python-format
msgid "Some unknown keys were provided: %(keys)s"
msgstr "Byly zadány neznámé klíče: %(keys)s"

#: venv/lib/python3.12/site-packages/django/contrib/postgres/validators.py:81
#, python-format
msgid ""
"Ensure that the upper bound of the range is not greater than %(limit_value)s."
msgstr "Ujistěte se, že horní hranice rozsahu není větší než %(limit_value)s."

#: venv/lib/python3.12/site-packages/django/contrib/postgres/validators.py:90
#, python-format
msgid ""
"Ensure that the lower bound of the range is not less than %(limit_value)s."
msgstr "Ujistěte se, že spodní hranice rozsahu není menší než %(limit_value)s."

#~ msgid "Nested arrays must have the same length."
#~ msgstr "Vnořená pole musejí mít stejnou délku."

#~ msgid "Map of strings to strings/nulls"
#~ msgstr "Mapování řetězců na řetězce či hodnoty NULL"

#, python-format
#~ msgid "The value of “%(key)s” is not a string or null."
#~ msgstr "Hodnota s klíčem \"%(key)s\" není řetězec ani NULL."
